### DISCUSSION TITLE
前置代理需要加快开发

### DISCUSSION THREAD

**Discussion by 2378** [SeaHOH/GotoX#223] [问答/咨询]
据反馈，目前CFW运行正常，但无法更新IP、HOST，部分网络环境已复现，已排除DNS污染可能。

基本可以肯定是 raw.githubusercontent.com 被墙

**Comment by SeaHOH** [comment]
是针对 IP 的间隙性干扰阻断，相关域名都受影响，已经这样好多年了。觉得麻烦的话，可以自行把这些域名都加入 CFW 代理规则。
